bare_metal_billing/billing.py

```diff
@@ -0,0 +1,129 @@
+import csv
+import sys
+import math
+from dataclasses import dataclass
+
+from bare_metal_billing import models
+
+
+BM_SU_LIST = [
+    "BM FC430",
+    "BM FC830",
+    "BM GPUA100SXM4",
+    "BM GPUH100",
+]
+
+RESOURCE_CLASS_2_SU_MAPPING = {
+    "lenovo-sd650nv2-a100": "BM GPUA100SXM4",
+    "sd650nv2": "BM GPUA100SXM4",
+    "fc430": "BM FC430",
+    "fc830": "BM FC830",
+}
+
+SU_RESOURCE_LIST = ["vCPUs", "RAM", "vGPUs"]
```

Comment:
Do we call it "vCPUs" everywhere of just "vCPU"? 

"vGPUs" is definitely incorrect, these are full physical GPUs. I don't remember what we called them in the nerc-rates code but we should probably be consistent. So, I guess either "GPUs" or "GPU" whatever we use for other repos.